usaspending_api/etl/transaction_loaders/fpds_loader.py

```diff
@@ -43,54 +41,53 @@ def delete_stale_fpds(date):
     provided detached_award_procurement_id list
     Returns list of awards touched
     """
-    if not date:
-        return []
```

Comment:
This adds some context to my last comment. I assume we already argued about this when the reloader was refactored the first time?

Reply:
I don't recall if we did. I moved it because there were too many return statements in this function. The logic is handled in get_deleted_fpds_data_from_s3() which is a tiny risk for future refactors